frontend/src/components/postViews/index.js

```diff
@@ -34,7 +33,7 @@ class PostViews extends Component {
       <div className={ styles.postViews }>
         <Icon iconName={ 'Eye' } className={ styles.viewIcon } />
         <p className={ styles.postViewsText }>
-          {`${unitFormatterUtil.getNumberSuffix(this.state.views)} 
+          {`${UnitFormatterUtil.getNumberSuffix(this.state.views)} 
                     ${this.state.views === 1 ? 'view' : 'views'}`}
```

Comment:
Strangely formatted code? Maybe use something like this:

```suggestion
          { `
              ${ UnitFormatterUtil.getNumberSuffix(this.state.views) } 
              ${ this.state.views === 1 ? 'view' : 'views' }
          `}
```